CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-## UNRELEASED
+*Feb 7, 2023*
+
+## v0.2.5
+
```

Comment:
seems not all [changes](https://github.com/crypto-com/pystarport/compare/v0.2.4...main) mentioned in the doc

Reply:
fixed